localization/english/bookmark/bookmark_605_l_english.yml

```diff
@@ -1,11 +1,34 @@
 
- #Othmar Garithos
- othmar_garithos_character:0 "Grand Marshal Garithos"
- othmar_garithos_character_desc:0 "PH"
+ #Luc Valonforth
+ valonforth_character:0 "Captain Luc Valonforth"
+ valonforth_character_desc:0 "Captain Luc Valonforth of the Alliance's first legion is a loyal officer of Lordaeron. As one of prince Arthas's former lieutenants, Valonforth followed his master in the expedition to Northrend against Mal'Ganis, to better end betrayed by the corrupted prince. Unlike his fellow captains and comrades, though, Luc Valonforth and his men were not slain by Frostmourne but abandonned and left stranded on the frozen shores of Northrend. With no way to return to Lordaeron, the remaining Alliance warriors held their position in Northrend, trying to survive. Now, with the Scourge weakened and the Illidari invasion of Northrend, opportunities may present to Valonforth to push back the undead and secure his position. One day, perhaps, ships will bring the forgotten legion back home... Are the abandoned soldiers doomed to death in the frozen north or will they be able to survive or even prosper in the cold wasteland ?"
+
+ #Kael'thas Sunstrider
+ kaelthas_character:0 "Prince Kael'Thas"
+ kaelthas_character_desc:0 "Crown-prince of Quel'Thalas and archmage of Dalaran, Kael'Thas Sunstrider lost both of his homes to Arthas and the Scourge during the Third War. With his kingdom destroyed and his father dead, Kael renounced the title of king but still gathered other elven survivors, renaming them Blood Elves in honor of their fallen brethren. Soon, the prince discovered that his people was not only threatened of extinction by the Scourge, but also by its own addiction to magic causing an insatiable thirst since the Sunwell, source of Quel'Thalas's power, was lost. Looking for allies, the prince first pledged himself to the Alliance, but desperation, magical thirst and Alliance ruler Garithos's hostility against the elves forced Kael'Thas to ally with the Illidari and be accused of treason by their former human allies. The rogue demon Illidan Stormrage offered Kael and his people new sources of magic in exchange for their loyalty, an offer that could not be ignored. Now in the service of Illidan, Kael'Thas is engaged along with his new master in a quest to destroy the Lich King, an opportunity to avenge the fall of Quel'Thalas. Engaged in a war with the Lich King, will Kael'Thas succumb to the darkness of the Scourge ? Will he be led to damnation by his pact with Illidan and the Legion ? Or will he bring glory to the blood elves again ? "
```

Comment:
```suggestion
 kaelthas_character_desc:0 "Crown-prince of Quel'Thalas and archmage of Dalaran, Kael'Thas Sunstrider lost both of his homes to Arthas and the Scourge during the Third War. With his kingdom destroyed and his father dead, Kael renounced the title of king but still gathered other elven survivors, renaming them Blood Elves in honor of their fallen brethren. Soon, the prince discovered that his people was not only threatened by extinction by the Scourge but also by its own addiction to magic causing an insatiable thirst since the Sunwell, the source of Quel'Thalas's power, was lost. Looking for allies, the prince first pledged himself to the Alliance, but desperation, magical thirst, and Alliance ruler Garithos' hostility against the elves forced Kael'Thas to ally with the Illidari and be accused of treason by their former human allies. The rogue demon Illidan Stormrage offered Kael and his people new sources of magic in exchange for their loyalty, an offer that could not be ignored. Now in the service of Illidan, Kael'Thas is engaged along with his new master in a quest to destroy the Lich King, an opportunity to avenge the fall of Quel'Thalas.\n\nEngaged in a war with the Lich King, will Kael'Thas succumb to the darkness of the Scourge? Will he be led to damnation by his pact with Illidan and the Legion? Or will he bring glory to the blood elves again?"
```

Reply:
^ this still stands, can be committed as is.